core/src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -507,4 +513,26 @@ protected static Fuzziness randomFuzziness(String fieldName) {
     protected static boolean isNumericFieldName(String fieldName) {
         return INT_FIELD_NAME.equals(fieldName) || DOUBLE_FIELD_NAME.equals(fieldName);
     }
+
+    private static class MockTermsLookupFetchService extends TermsLookupFetchService {
```

Comment:
You can still override only one method when you extend a class. Anyways, keep the service, it's fine